extensions/rich_text_components/Image/Image.py

```diff
@@ -47,9 +48,19 @@ class Image(base.BaseRichTextComponent):
         'default_value': '',
     }, {
         'name': 'alt',
-        'description': 'Alternative text (for screen readers)',
+        'description': (
+            'Briefly explain this image to a visually impaired '
+            'learner'),
         'schema': {
             'type': 'unicode',
+            'validators': [{
+                'id': 'is_nonempty',
+            }],
+            'ui_config': {
+                'placeholder': (
+                    'Description of Image (Example : "George Frideric, '
```

Comment:
@kirankonduru I really like what you did with the descriptive text! I think that makes the example very clear. The only nit I would mention is that the composer's name is George Frideric Handel, so just add Handel to the end of his name.